### ISSUE TITLE
Are you still in business? how to get Authorization Code

### ISSUE THREAD

**Issue by Fletch5** [etech-software/dongle-emulator#2]
Are you still operational as a company? I have sent emails stating I need help with Authorizing a new computer that we are moving our software over to windows 10.
I also paid for cloud service a few days ago, and have not received any response.

Help Please?

**Comment by ackurdeeve** [comment]
I have found a solution.

https://etechvirtualization.com/dongleuser/users/

**Comment by tRImAnTEoUc** [comment]
And now that site is not working anymore. (https://etechvirtualization.com/) 
I believe he has abandoned this project?

**Comment by ackurdeeve** [comment]
Yes, can not open it anymore. I do not know what happened. Maybe need to check a few days later.

**Comment by rondbe** [comment]
Need authorization code - paid for this but unable to access.  For customers who have paid for permanent use, appreciate if the owner could provide an alternative backup pro software WITHOUT any authorization code.

**Comment by ackurdeeve** [comment]
Seems they closed the auth server, I can not activate my dongle using the backup auth code .

**Comment by macanal** [comment]
El mismo problema. cambie procesador de equipo y ahora me pide código de validación con u nuevo requeset codigo.
Llevo varios años con este sistema y siempre funciono perfectamente. Creo que los usuarios tendríamos que tener acceso al generador de códigos.

**Comment by ackurdeeve** [comment]
I now do not expect the auth code will work anymore. Hope somebody can reverse engineering this dongle emulator.

**Comment by macanal** [comment]
A ver si alguien es capaz a buscar una solución, la verdad que funcionaba muy bien y creo que somos muchos usuarios. Pensaba que seria cuestión de saber con que software se generaban los códigos. Si los creadores no vuelven a aparecer online.

**Comment by ackurdeeve** [comment]
The dbpro uses C# as its UI, however, most of the functions are in dbpro.dll. I think it is writen using C++. There is several enrypt algorithms in it, not very complicate. I used ida64 to see it, if you have dng file, I think it is possible to make it work again. For new computers, you have to know the registion mechanism, not easy, it has a 256 table verify the authcode.

**Comment by ackurdeeve** [comment]
I have just create a channel using Telegram. 

https://t.me/donglebackuppro

**Comment by macanal** [comment]
Muchas gracias por la ayuda

**Comment by Jdorland78** [comment]
Hey Iogangr54, with sentineldriver 2010 it's also possible but has the same problem as Multikey, not I signed driver. I was in contact with VIPdongle but they ask arround €500,- for an emulated dongle but don't provide a test version so the risk is to high (for me). 
I'm stuck with this problem, The work of Ackurdeeve is impressive but I can't build it further.

**Comment by NoCloud292** [comment]
Hi Jdorland78 the HASPHL2010 work on Windows 10 but with signed driver disabled. With cloud authorisation code not available is a very problem.
